crucible-llvm/src/Lang/Crucible/LLVM/MemModel.hs

```diff
@@ -395,7 +395,8 @@ evalStmt sym = eval
              | Just Refl <- testEquality handleTp expectedTp -> return (HandleFnVal h)
              | otherwise -> failedAssert
                  $ unwords ["Expected function handle of type " <> show expectedTp
-                           ,"but found handle of type " ++ show handleTp]
+                           ," for call to function " <> show (handleName h)
```

Comment:
The `unwords` above is also going to add spaces between these list elements, did you intend to have two spaces here?